server/src/main/proto/set_pb.proto

```diff
@@ -25,7 +25,36 @@ message SetGetResponse {
     repeated string values = 2;
 }
 
+message SetDeleteRequest {
+    required string key = 1;
+    required string entity = 2;
```

Comment:
Add a comment :
`The entity that will be deleted from the set`.

Reply:
Done.